frontend/src/container/NotificationWall.jsx

```diff
@@ -186,6 +291,12 @@ export class NotificationWall extends React.Component {
         <div className='notification__list'>
           {props.notificationPage.list.length !== 0 && props.notificationPage.list.map((notification, i) => {
             const notificationDetails = this.getNotificationDetails(notification)
+            const icons = notificationDetails.icon.split('+')
+            const icon = (
+              icons.length === 1
+                ? <i className={`fa fa-fw fa-${icons[0]}`} />
```

Comment:
Before there was a css class (`.notification__list__item__icon`) in the icone, was it removed intentionally?

Reply:
Yes! :-)
Not needed anymore. I use a CSS selector that matches both this icon and the ComposedIcon using the `fa` class.

Reply:
And this will not affect other icons in the notification wall (like the circle that marks a message not read)? You have `mainIconCustomClass` and `smallIconCustomClass` at ComposedIcon if you need

Reply:
Indeed. I'll added a first-child selector to make it specific.